apps/aecore/src/aec_conductor.erl

```diff
@@ -409,8 +425,9 @@ dispatch_worker(Tag, Fun, State) ->
         false ->
             Wrapper = wrap_worker_fun(Fun),
             {Pid, Ref} = spawn_monitor(Wrapper),
+            {ok, TRef} = timer:exit_after(_TODO=200, Pid, shutdown),
             State1 = block_tag(Tag, State),
-            Workers = orddict:store(Pid, {Tag, Ref}, State1#state.workers),
+            Workers = orddict:store(Pid, {Tag, Ref, TRef}, State1#state.workers),
```

Comment:
I think it would make sense to introduce a record for a worker?! Three parameters in a tuple is stretching my mind :-)

Reply:
Looks good so far 👍

Reply:
record makes sense

Reply:
record introduced